mkdocs.yml

```diff
@@ -72,6 +72,7 @@ nav:
         - Networking: tutorials/networking.md
         - Frameworks:
           - Dungeon Framework Reference: tutorials/dungeon_framework_reference.md
+        - Weapons: tutorials/weapons.md
```

Comment:
old name?

Reply:
Renamed the tutorial to weapons_tutorial. Trying to stick to the naming conventions that references are just the topic, and tutorials get "tutorial" in the filename.